### PR TITLE
Made traitor kevlar/armor padding accessories examine-proof.

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
They are a boring, lazy 2 tc investiment that only offer an edge to the player's armor at the cost of being valid should they examine your mob beyond flavor texts.
Stealthy uplink items are also junk, so it's a little improvement.

## Why It's Good For The Game
QoL stealthing. There are better 2 tc investments around.

## Changelog
:cl:
tweak: Attached kevlar/soft/plastic padding accessories are now stealthier and will no longer be displayed on mob examine.
/:cl:

### CODE REVIEW

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -1332,21 +1332,21 @@ datum/uplink_item/stealthy_weapons/taeclowndo_shoes
...
-	desc = "Kevlar sheets to add to jumpsuit to help against bullets and melee."
+	desc = "Inconspicious kevlar sheets worn underneath jumpsuits to protect the user against bullets and melee."
 	item = /obj/item/clothing/accessory/kevlar
 	cost = 2
 	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
 
 /datum/uplink_item/suits/plastic
 	name = "Plastic sheet"
-	desc = "Plastic body sheet to add to a jumpsuit to help against laser and energy harm."
+	desc = "Inconspicious  plastic body sheet worn underneath jumpsuits to protect the user against laser and energy beams."
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm]: This... Defeats the reason to have the plastic sheet...

Comment (Ghommie) [reply code/modules/uplink/uplink_items.dm]: ok fine I'll recall it to energy dampening. whatever soon.

Comment (Trilbyspaceclone) [reply code/modules/uplink/uplink_items.dm]: The way plastic protects you against laser is that when it burns the top layer of the plastic the thick smoke absorbs a lot of energy.

Comment (Putnam3145) [reply code/modules/uplink/uplink_items.dm]: is it explicitly ablative?

EDIT: doesn't say it's ablative, so we can just chalk this up to spess magic tbh, no reason to have every piece of materiel in the game be Fully Realistic

**Review by silicons — APPROVED** [review]

**Pull request merged**